Fix/mudanca imagem role ubuntu

afernandescneto opened the pull request:
# Utiliza Ubuntu como imagem padrão

Foi realizada a mudança da imagem da role. 

- [X] Garanta que seu **topic/feature/bugfix branch** tenha uma branch nomeada e não a sua branch main esteja no PR
- [X] Dê um titulo que expresse o objetivo do PR
- [X] Associe seu PR a uma Issue criada no repositósito. Caso seja uma correção de linguagem ou pequenas correções, não é necessário
- [X] Descreva o objetivo do PR
- [X] Inclua links relevantes para a sua modificação/sugestão/correção
- [X] Descreva um passo-a-passo para testar o seu PR

## Issue

closes #8
closes #11 

## Objetivo

O template estava usando CentOS como imagem padrão, mas estava sendo modificada sempre que usado para Ubuntu. Essa mudança faz com que use o Ubuntu por padrão. 

## Referências

Usado como base para referência da imagem:
https://github.com/mentoriaiac/iac_role_nginx/blob/main/molecule/default/molecule.yml

## Como testar
Usar o seguinte comando:
```bash
molecule test
```
<!--
Marque um `x` dentro de [ ] para os itens que você forneceu informação
Para modificar este template no seu repositório, basta criar o arquivo .github/pull_request_template.md nele.
-->

afernandescneto committed:
Correção do nome da role para adequar ao modo de usar do README

afernandescneto committed:
Correção do gerenciamento de dependências do Python criando lock do requirements com pip freeze]

afernandescneto committed:
fix: Mudança da imagem para Ubuntu(20.04) que será usado pela role.

Signed-off-by: Antonio Fernandes Coelho Neto <afernandescneto@gmail.com>

Co-authored-by: cflb <iamcflb@gmail.com>
Co-authored-by: Marcos Medeiros <soulinux@gmail.com>

gomex commented on requirements.txt:
```diff
@@ -1,8 +1,56 @@
 ansible==2.10.5
```
O que acha de aproveitar e atualizar logo para os pacotes mais novos? 

O pipeline está quebrado também.

afernandescneto replied:
Realmente. Focamos na resolução e nem nos atentamos a essa possibilidade. Vou verificar com Cleyton (cflb) para resolvermos isso.

gomex replied:
Uma solução pode ser pegar esse requirements.txt e usar aqui: https://github.com/mentoriaiac/iac_role_consul/blob/main/requirements.txt

lgfa29 replied:
Atualizado para 5.6.0.

gomex reviewed (changes requested):
Gostei das mudanças, mas pensei que talvez possamos usar essa chance pra atualizar os pacotes do python.

lgfa29 committed:
chore: atualiza dependências e adiciona licença

lgfa29 committed:
ci: atualiza versão dos runners para ubuntu 20.04

Atualizando versão dos runners para utilizar a versão 20.04 do Ubuntu e
não atualizar o pacote de setuptools para ficar igual o teste de lint
que conseguiu instalar a versão 5.6.0 do Ansible.

lgfa29 committed:
chore: definindo um valor para autores não-padrão

O ansible-lint verifica se o valor padrão da chave author do
meta/main.yml foi sobreescrita.

Esse commit define um valor diferente do padrão.

lgfa29 committed:
chore: adictiona dependências de Linux

Quando rodando `pip install -r requirements.txt` em uma máquina Linux,
algumas dependências a mais são instaladas.

lgfa29 committed:
chore: manutenção de alguns arquivos de texto

Remove alguns espaços em branco e adiciona quebra de linha em arquivos
que estavam incorretos.

lgfa29 committed:
fix: arrumando nome da variável na matriz de teste

lgfa29 committed:
fix: usar codenome para as versões de Ubuntu

A API do Ansible Galaxy usa os codenomes do release para identificar as
versões do Ubuntu.

Fonte: https://galaxy.ansible.com/api/v1/platforms/?search=ubuntu

lgfa29 committed:
ci: muda a cor de mensagens de debug do Ansible

A cor padrão de mensagens de debug nos logs do GitHub Action ficam
difíceis de ler por causa do fundo escuro.

lgfa29 committed:
fix: downgrade do molecule para evitar login bug

gomex reviewed (approved):
LGTM
